src/en/space-station-14/round-flow/proposals/station-ai.md

```diff
@@ -0,0 +1,162 @@
+- Cargo bounty - Call it “Bluespace Calculation” or any other fancy name. Basically the same as research but also gives out something that can fulfil a cargo bounty. Giving it to cargo could be done through integration with the [paperwork management UI](#assorted-ideas)? Should be separate from research resource sink to not let AI just double-dip.
+
+#### Routines (Momentary actions)
+
+Things that don’t require sustained cost, but should not be completely free. Practically anything that interacts with things beyond a digital interface.
+
+- Opening a door! Minimal amount required.
+- Speech when the radio process is inactive. A somewhat substantial amount when surviving off a single or two AI servers, but not extremely hard hitting. Binary channel is always free however.
+- Left-click or alt-click on vents to unlock them, for cheap
+- **TODO**. Ideas: sensor sweep? Mass applying (un)bolting to everything an APC’s connected to? Atmospherics reset action that can be used from the action bar/alt-click radial menu and applied to an atmos alarm to reset it to default? Clicking vents to unlock them?
+- Consoles! They aren’t free and cut a decent chunk, to encourage the AI player to choose a Compute option, but in a pinch this should work too. It also gets paid by the player having to manually locate a console and being unable to check on things elsewhere but the vicinity of the console.
+
+### Sources
+
+- AI specialized machinery, contributing a significant amount of processing capability to the AI.
+  - The downside to this is that it requires someone physical to build it, and needs a decent amount of resources
+  - Such machinery should be difficult to destroy or even move. It stays where it stays, unless the situation requires it and someone puts enough effort to move it; or break it.
+  - Generates heat up to a throttle cutoff, and works better when cooled. When not in an optimal environment, provides less computing power, but still comparable to servers.
+  - Backup power? Camera built in?
+- Crew devices; computers, machines, servers
+  - These don’t contribute as much **Compute** as they are not specialized for AI use. Servers do provide more **Compute** than just consoles, and provide **Bandwidth** in amounts comparable or identical to dedicated machines..
+  - Locks out the device for crew use, rendering it inoperable. This acts as a conflict point with the crew. Computers don’t work, machines either. Servers don’t provide normal functionality or do so in a very limited way.
+    - E.g. crew monitoring server could give invalid/partial data with a warning, or just not work at all. Same for telecom servers.
```

Comment:
I'd like to think of the Malf AI slowly infecting machines as like a virus. The little APCs and pointless vending machines hacked don't provide much compute or bits by themselves, but all together, it turns into a *very big* problem for the crew, and a sweet benefit to Malf's compute and bits.